platform/nservicebus.include.md

```diff
@@ -1,3 +1,3 @@
-NServiceBus, the heart of the system, is a messaging and workflow framework that helps create distributed systems that are scalable, reliable and easy to modify. It supports various messaging patterns, handles long-running business processes in the form of [sagas](/nservicebus/sagas), and provides abstraction over multiple [queuing technologies](/transports/). While most queuing technologies try to make guarantees regarding 'at least once' or even 'exactly once' delivery, they often fall short of this promise. NServiceBus contains mechanisms to automatically solve intermittent delivery problems by retrying messages and falling back to an error queue where they can be exposed by the rest of the Platform for human intervention ([ServiceControl](/servicecontrol/), [ServicePulse](/servicepulse/)).
+NServiceBus, the heart of the system, is a messaging and workflow framework that helps create distributed systems that are scalable, reliable and easy to modify. It supports various messaging patterns, handles long-running business processes in the form of [sagas](/nservicebus/sagas), and provides an abstraction over several [queuing](/transports/) and [storage](/persistence/) technologies. While most queuing technologies try to make guarantees regarding 'at least once' or even 'exactly once' delivery, they often fall short of this promise. NServiceBus contains mechanisms to automatically solve intermittent delivery problems by retrying messages and falling back to an error queue where they can be exposed by the rest of the Platform for human intervention ([ServiceControl](/servicecontrol/), [ServicePulse](/servicepulse/)).
 
-Moreover, NServiceBus is thoroughly extensible and can be tailored for many requirements. It is compatible with several technologies and many elements of the system can be replaced with custom implementations.
+Moreover, NServiceBus is thoroughly extensible and can be tailored for many requirements. It is compatible with several technologies and many elements of the system can be replaced with custom implementations.
```

Comment:
I'd prefer to avoid using "moreover" if possible. The rest of the sentence sounds a bit too general. Maybe:

```suggestion
NServiceBus is modular in design and highly extensible. In addition to the supported options for queueing, storage, serialisation, logging, and more, many elements of the system can be replaced with custom implementations for specific scenarios.
```

BTW - this paragraph already seems to be live on https://docs.particular.net/platform/ 🤔

Reply:
> BTW - this paragraph already seems to be live

That's because it was in the original document. I would never use the word "moreover" when words "and" or "also" or "" exist.